### ISSUE TITLE
VMUI: Hung when use Cmd + V for pasting with autocomplete

### ISSUE THREAD

**Issue by yuriydzobak** [VictoriaMetrics/VictoriaMetrics#5923]
### Describe the bug

The browser crashes/stacks when a large query is pasted in the Query field with autocomplete enabled

### To Reproduce

1. Open https://play.victoriametrics.com
2. Enable auotcomplate
3. run `{__name__!=""}` and find a big query with labels
3. Past query via ctrl+ V or cmd + V
```
ALERTS_FOR_STATE{alertgroup="vmcluster", alertname="ConcurrentInsertsHitTheLimit", cluster="sandbox", container="vmagent", dc="sandbox-gke", endpoint="http", env="sandbox", instance="10.71.5.2:8429", job="vmagent-aggregator", namespace="streamagg-benchmark", pod="vmagent-aggregator-6576fcb566-68dnn", prometheus="monitoring/monitoring-vmagent", service="vmagent-aggregator", severity="warning", vm_account_id="0", vm_project_id="0"} > 0 
```
4. The page was hung/crashed

### Version

reproduced on cluster version 1.98/1.99 versions

**Comment by hagen1778** [comment]
cc @Loori-R

**Comment by valyala** [comment]
This bug has been fixed in [v1.100.0 release](https://docs.victoriametrics.com/changelog/). Closing the issue as fixed.
